app/models/case_log.rb

```diff
@@ -15,13 +15,29 @@ def validate_property_number_of_times_relet(record)
     end
   end
 
+  def validate_reasonable_preference(record)
+    if record.homelessness == "No" && record.reasonable_preference == "Yes"
+      record.errors.add :reasonable_preference, "can not be Yes if Not Homesless imediately prior to this letting has been selected"
```

Comment:
Typo on immediately

Reply:
Also on homeless

Reply:
Is there a Grammarly extension for vs code 😂 I'll fix this now